modules/internal/ChapelArray.chpl

```diff
@@ -2964,7 +2860,7 @@ module ChapelArray {
   }
 
   proc =(ref a: domain, b: _tuple) {
-    a._value.clearForIteratableAssign();
+    a.clear();
```

Comment:
Can you explain why you made this change / why clearForIterableAssign does not need to be called here?